assets/src/blocks/Spreadsheet/SpreadsheetEditor.js

```diff
@@ -52,26 +54,24 @@ export const SpreadsheetEditor = ({
             }}
           />
           <div className="sidebar-blocks-help">
-            <ul>
-              <li>
-                {/* eslint-disable-next-line no-restricted-syntax, @wordpress/i18n-no-collapsible-whitespace */}
-                { __(`From Your Google Spreadsheet Table choose File -> Publish on web.
-                No need to choose the output format, any of them will work.
-                A pop-up window will show up, click on the Publish button and then OK when the confirmation message is displayed.
-                Copy the URL that is highlighted and paste it in this block.`, 'planet4-blocks-backend') }
-              </li>
-              <li>
-                {/* eslint-disable-next-line no-restricted-syntax, @wordpress/i18n-no-collapsible-whitespace */}
-                { __(`If you make changes to the sheet after publishing
-                  then these changes do not always immediately get reflected,
-                  even when "Automatically republish when changes are made" is checked.`, 'planet4-blocks-backend') }
-              </li>
-              <li>
-                {/* eslint-disable-next-line no-restricted-syntax, @wordpress/i18n-no-collapsible-whitespace */}
-                { __(`You can force an update by unpublishing and republishing the sheet.
-                  This will not change the sheet's public url.`, 'planet4-blocks-backend') }
-              </li>
-            </ul>
+            <p>
```

Comment:
This is a quick fix to remove an ugly padding:

- Before
<img width="273" alt="Screenshot 2025-04-18 at 17 25 35" src="https://github.com/user-attachments/assets/a4948cbe-4421-42f4-80d2-b37aa025cd0a" />

- After
<img width="272" alt="Screenshot 2025-04-18 at 17 25 42" src="https://github.com/user-attachments/assets/6ca5562e-c471-455f-bdb3-f042902da856" />